datafusion/physical-expr/src/functions.rs

```diff
@@ -1379,95 +1370,6 @@ mod tests {
             Utf8,
             StringArray
         );
-        test_function!(
```

Comment:
For the following tests, some are migrated to `strpos()`, and some are removed because tests of `strpos()` already have them covered